code/game/gamemodes/devil/true_devil/_true_devil.dm

```diff
@@ -67,7 +67,8 @@
 	//Left hand items
 	for(var/obj/item/I in held_items)
 		if(!(I.flags_1 & ABSTRACT_1))
-			if(I.blood_DNA)
+			GET_COMPONENT_FROM(FR, /datum/component/forensics, I)
```

Comment:
you do this exact two pattern line so many times you should make it a macro or something